server/conf/evolutions/default/82.sql

```diff
@@ -0,0 +1,8 @@
+# --- !Ups
+
+ALTER TABLE applications ADD COLUMN IF NOT EXISTS eligibility_determination VARCHAR;
+
+UPDATE applications SET eligibility_determination = 'NOT_COMPUTED' WHERE eligibility_determination IS NULL;
```

Comment:
(question) Will this column be made `not null` at some point?

Reply:
I think the column should probably be `NOT NULL` from the start with a default value of `NOT_COMPUTED`

Reply:
Agreed